### PR TITLE
Auv added a doc splitter

### PR DESCRIPTION
**Pull request by marsbroshok**

### CODE REVIEW

File: general/document_splitter.ipynb [hunk 1]
```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "W-dlkukAsGuQ"
+   },
+   "source": [
+    "# Overview\n",
```
Comment (camushoilingma) [inline general/document_splitter.ipynb:9]: Do we need all this intro to document AI here? We can just put in README in the top directory or something to stay consistent with other notebooks.

File: general/document_splitter.ipynb [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    "\n",
```
Comment (camushoilingma) [inline general/document_splitter.ipynb:10]: Maybe the title should be "Document AI General Form Splitter" to stay consistent

File: general/document_splitter.ipynb [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+    "Now, you're ready to use the Document AI API!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LPZhvzwesGvB"
+   },
+   "source": [
+    "# Document Splitter (Private Beta)"
```
Comment (camushoilingma) [inline general/document_splitter.ipynb:86]: Remove "Private Beta" or this entire section of text. Replace with a simple title "Set your Processor Variables" as done in the other two notebooks.

File: general/document_splitter.ipynb [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO(developer): Fill these variables with your values before running the sample\n",
+    "project_id= 'YOUR_GCP_PROJECT_ID'\n",
```
Comment (camushoilingma) [inline general/document_splitter.ipynb:108]: make these all caps like in the other notebooks.

**Review by camushoilingma — CHANGES_REQUESTED** [review]
Looks good, just some changes to mirror the other notebooks.

File: general/document_splitter.ipynb (hunk 1 above)
Comment (marsbroshok) [reply general/document_splitter.ipynb]: agree, done

File: general/document_splitter.ipynb (hunk 2 above)
Comment (marsbroshok) [reply general/document_splitter.ipynb]: Changed to "# Document AI General Document Splitter" - "document" is twice there but that's the naming :)

File: general/document_splitter.ipynb (hunk 3 above)
Comment (marsbroshok) [reply general/document_splitter.ipynb]: done

File: general/document_splitter.ipynb (hunk 4 above)
Comment (marsbroshok) [reply general/document_splitter.ipynb]: done

**Review by camushoilingma — APPROVED** [review]

**Pull request merged**